spec/graphql/queries/book_query_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Types::QueryType do
+  describe 'query book' do
+    it 'can query a single book' do
+      short_history = create(:book, id: 5, isbn: "9780307885166", title: "A Short History Of Nearly Everything", author: "Bill Bryson", image_url: "url", page_count: 876, summary: "This new edition")
```

Comment:
very minor nit: If the only reason you are overriding factoryBot's random fields is so that you can assert the exact value of each of these fields, it's totally fine to just assert on datatypes OR I would just use a real `Book.create` to keep the exact assertions you have.

Reply:
Changed to assert on datatypes instead! Thank you